src/werkzeug/routing/rules.py

```diff
@@ -454,7 +454,7 @@ def __init__(
         websocket: bool = False,
     ) -> None:
         if not string.startswith("/"):
-            raise ValueError("urls must start with a leading slash")
+            raise ValueError("urls must start with a leading slash:", string)
```

Comment:
This is not how exception class constructors work.

Reply:
oops, thats embarrassing. fixed now